internal/services/storage/storage_account_local_user_resource.go

```diff
@@ -93,7 +92,7 @@ func (r LocalUserResource) Arguments() map[string]*pluginsdk.Schema {
 					"key": {
 						Type:             pluginsdk.TypeString,
 						Required:         true,
-						ValidateFunc:     computevalidate.SSHKey,
+						ValidateFunc:     validate.SSHKey,
```

Comment:
Have you verified that the storage account local user still doesn't support ED25519 public key yet?

Reply:
Actually it supports ed25519. I have reverted this diff so now the PR covers both compute and storage account local user. Commit ref: dfa49be2f5